src/index.js

```diff
@@ -0,0 +1,8 @@
+
+function say(name){
+    if(name!=null)
+        return "Hello " + name + "!";
+    else return "Hello world!";
+}
+
+module.exports = {say:say}
```

Comment:
```suggestion
module.exports = {say}
```

Missing newline at eof.